.github/workflows/ci.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     - run: rustup toolchain install stable --profile minimal
     - uses: Swatinem/rust-cache@v2  
     - name: Install SP1 toolchain
-        run: |
```

Comment:
@fakedev9999, looks like the "Cargo Tests" were skipped because of this error. Now I see the [same error](https://github.com/succinctlabs/op-succinct/actions/runs/18739873750/job/53453789338?pr=657#step:8:2244) popping up in CI that I was talking about earlier when running things locally.